python/data/chap3/README.rst

```diff
@@ -20,6 +20,33 @@ start.dlis.part              SUL + all the VR and LRSH bytesiBytes 50 and 51
 /reprcode                    Object attribute of various representation codes
 ============================ =================================================
 
+implicit folder contains full files and it's worth to describe it in more
```

Comment:
> and it's worth to describe it in more details.

Skip this? Its obvious that its worth describing as we have a full table below
(table is great btw